google/cloud/spanner/value.h

```diff
@@ -305,20 +305,11 @@ class Value {
     return GetValue(T{}, std::move(value_), type_);
   }
 
-  /**
-   * Allows Google Test to print internal debugging information when test
-   * assertions fail.
-   *
-   * @warning This is intended for debugging and human consumption only, not
-   *   machine consumption as the output format may change without notice.
-   */
-  friend void PrintTo(Value const& v, std::ostream* os);
-
   /**
    * Outputs string representation of a given Value to the provided stream.
    *
    * @warning This is intended for debugging and human consumption only, not
-   *   machine consumption as the output format may change without notice.
+   *     machine consumption as the output format may change without notice.
```

Comment:
Nit: Now that you're editing this, you might as well put a comma after "consumption".

Reply:
Done.